Source/WebCore/inspector/InspectorNodeFinder.cpp

```diff
@@ -91,8 +91,8 @@ void InspectorNodeFinder::searchUsingDOMTreeTraversal(Node& parentNode)
         case Node::ELEMENT_NODE:
             if (matchesElement(downcast<Element>(*node)))
                 m_results.add(node);
-            if (is<HTMLFrameOwnerElement>(downcast<Element>(*node)))
-                performSearch(downcast<HTMLFrameOwnerElement>(*node).contentDocument());
+            if (RefPtr frameOwner = dynamicDowncast<HTMLFrameOwnerElement>(*node))
+                performSearch(frameOwner->contentDocument());
```

Comment:
However, this should call `performSearch(frameOwner->protectedContentDocument().get());`, adding a  `protectedContentDocument()` getter to HTMLFrameOwnerElement that returns a RefPtr.